pkg/cover/report.go

```diff
@@ -74,7 +74,20 @@ type line struct {
 	progIndex int          // example program index that covers this line
 }
 
-func (rg *ReportGenerator) prepareFileMap(progs []Prog) (map[string]*file, error) {
+func coverageCallbackMismatch(debug bool, numPCs int, unmatchedProgPCs map[uint64]bool) error {
+	debugStr := ""
+	if debug {
```

Comment:
Looking at it with a fresh mind -- do you have any other debugging output in mind?

If it's only about the list of exact unmatched PCs, maybe it's better to just always print them all in the case of an error? How many bad PCs did you see locally?

Reply:
On ci-upstream-gce-arm64 there are 440219 of them :)

Reply:
Ah, ok. That's many.